lib/definition_parser.rb

```diff
@@ -0,0 +1,48 @@
+class DefinitionParser
```

Comment:
should we name this class DeploymentDefinitionParser ?

Reply:
also what does this function actually do? convert from text to hash?

Reply:
> should we name this class DeploymentDefinitionParser?

Noted.

> also what does this function actually do? convert from text to hash?

Based on [this explanation](https://github.com/pathfinder-cm/pathfinder-mono/pull/43#issuecomment-600389796), it renders Pathfinder script in **deployment definition** to container attributes.